### PR TITLE
add systemd and rpm support

### PR DESCRIPTION
**Pull request by jjptapia**
added systemd and rpm for CentOS/redhat 7.x

### CODE REVIEW

File: packaging/rpm/rb_monitor.spec [hunk 1]
```diff
@@ -0,0 +1,49 @@
+Name:    rb_monitor
+Version: %{__version}
+Release: %{__release}%{?dist}
+
+License: GNU AGPLv3
+URL: https://github.com/redBorder/rb_monitor
+Source0: %{name}-%{version}.tar.gz
+
+BuildRequires: gcc librd-devel net-snmp-devel json-c-devel librdkafka-devel libmatheval-devel libpcap-devel
```
Comment (eugpermar) [inline packaging/rpm/rb_monitor.spec]: libpcap-devel?

File: rb-monitor.init.upstart [hunk 2]
```diff
@@ -0,0 +1,24 @@
+description "rb-monitor service" 
+author      "jjprieto@redborder.com" 
+
+start on filesystem or runlevel [2345]
```
Comment (eugpermar) [inline rb-monitor.init.upstart:4]: Does it have sense to talk about runlevels on systemd?

File: packaging/rpm/rb_monitor.spec (hunk 1 above)
Comment (eugpermar) [inline packaging/rpm/rb_monitor.spec:9]: We don't need libpcap, I will delete it from configure.rb_monitor and it should be removed from here too

File: rb-monitor.init.upstart (hunk 2 above)
Comment (jjptapia) [inline rb-monitor.init.upstart:4]: Not needed. This is only a base systemd service thinking in a supervise as chef that command the start or stop of the service.

File: packaging/rpm/rb_monitor.spec (hunk 1 above)
Comment (jjptapia) [inline packaging/rpm/rb_monitor.spec:9]: ok, let me check it.

**Pull request merged**